lib/tasks/rm_future_learn_from_pathways.rake

```diff
@@ -0,0 +1,105 @@
+namespace :rm_future_learn_from_pathways do
+  desc 'one-off task to remove the future learn courses from their pathways'
+  task remove: :environment do
+    puts 'Destroying pathway join records for all future learn courses'
```

Comment:
Happy to stand corrected if I'm missing something, but because pathways are predominantly a meta thing, we found it safest to just remove them all and run the seeds manually, like this:

https://github.com/NCCE/teachcomputing.org/blob/main/lib/tasks/fix_pathway_activities.rake#L6

Reply:
Thank you!

As a habit, I don't normally destroy everything and I didn't look at your 'fix' task as I didn't have anything to fix (yet) 😂 . That rake task looks like it will do what I need, so I'll delete the new one.

That means our recovery plan will require a rollback and running that task again.